.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/llvm/llvm-project.git
 [submodule "externals/stablehlo"]
 	path = externals/stablehlo
-	url = https://github.com/openxla/stablehlo.git
+	url = https://github.com/shark-infra/stablehlo.git
```

Comment:
What's the reasoning for this?

Reply:
I have the same confusion about this.

Reply:
Wait, I'm confused and did something wrong. Let me revert this.

Reconstructing what I think was going on here: my local stablehlo git repo was reporting a commit not found and that was because my repository had a remote of a throwaway fork of stablehlo that I had used locally when doing the switch from mhlo. I saw that it was the old fork and updated, thinking that I had accidentally committed the use of the throwaway fork. I switched it to the fork we use for patching on another project, thinking I needed to land some patches on the branch.

In any case, unintentional sequence of events matching work I was doing on a peer project. I apologize -- the change was not intentional (and that repo it is pointing to is a pure mirror of stablehlo).

Reply:
Resetting it here: https://github.com/llvm/torch-mlir/pull/2477/files